skycultures/arabic_lunar_stations/description.ar.utf8

```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html lang="ar" dir="rtl">
+<head>
+<title>منازل القمر العربية</title>
+</head>
```

Comment:
By formal rules no need header block for HTML'ed description, because we includes the descripting in our HTML page.